app/src/main/java/org/eyeseetea/malariacare/database/utils/planning/PlannedServiceBundle.java

```diff
@@ -0,0 +1,39 @@
+package org.eyeseetea.malariacare.database.utils.planning;
+
+import org.eyeseetea.malariacare.database.model.OrgUnit;
+import org.eyeseetea.malariacare.database.model.Program;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by idelcano on 08/08/2016.
+ */
+public class PlannedServiceBundle {
+    List<PlannedItem> plannedItems = new ArrayList<>();
+    List<OrgUnit> orgUnits;
+    List<Program> programs;
+
+    public PlannedServiceBundle(){
```

Comment:
@idelcano I don't think this constructor is needed (unless you are thinking about adding something else later on). Just initialise the variables. Also the variables should be private if we have getter/setter.